src/app/content/components/Toolbar.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { SFC } from 'react';
+import styled from 'styled-components';
+import {ListOl} from 'styled-icons/fa-solid/ListOl';
+import {Search} from 'styled-icons/fa-solid/Search';
+import {Print} from 'styled-icons/fa-solid/Print';
+import theme from '../../theme';
+import {bodyCopyRegularStyle} from '../../components/Typography';
+
+const iconStyles = {
+  mobile: {
+    height: '1.6rem',
+    width: '1.6rem',
+  },
+}
+
+const ListIcon = styled(ListOl)`
+  height: ${iconStyles.mobile.height};
+  width: ${iconStyles.mobile.width};
+  color: ${theme.color.secondary.darkBlue.base};
+  margin-right: 0.7rem;
+
+  :hover {
+    color: ${theme.color.secondary.darkBlue.foreground};
+  }
+
+`;
+
+const SearchIcon = styled(Search)`
+  height: ${iconStyles.mobile.height};
+  width: ${iconStyles.mobile.width};
+  color: ${theme.color.secondary.darkGray.base};
+  margin-right: 0.7rem;
+`;
+
+const PrintIcon = styled(Print)`
+  height: ${iconStyles.mobile.height};
+  width: ${iconStyles.mobile.width};
+  color: ${theme.color.secondary.darkGray.base};
+  margin-right: 0.7rem;
+
+`;
+
+const TopBar = styled.div`
+  height: 5rem;
+  max-width: 117rem;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (max-width: ${theme.mobileBreakpoint.default.width}) {
+    height: 4rem;
+  }
+  
+`;
+
+const ToCButton = styled.h3`
+  ${bodyCopyRegularStyle};
+  color: ${theme.color.secondary.darkBlue.base};
+  cursor: pointer;
+  margin: 0;
+
+  :hover {
+    color: ${theme.color.secondary.darkBlue.foreground};
```

Comment:
can we organize these so all the ToCButton, ListIcon and ToCButtonWrapper are next to each other? also maybe the `color` and hover state should be defined in ToCButtonWrapper? currently when you hover over the text the icon doesn't change, and the reverse, they should change together